items.js

```diff
@@ -35,4 +35,9 @@ const items  = [
     {name: 'microtransactions in mobile games', isWaste: true},
     {name: 'minecraft 3ds', isWaste: true},
     {name: 'drugs', isWaste: 'custom', custom: 'depends', explanation: 'im not qualified to tell this to you speak to your doctor or something'}
+    {name: 'scratch', isWaste: 'false', explanation: 'Come on! Scratch is awesome and its totally free!'}
+    {name: 'github', isWaste: 'free'}
+    {name: 'github teams', isWaste: 'custom', explantion: 'it has free a option and if you pick that the no but if you pay for it yes'}
+    {name: 'github PRO', isWaste: 'true'}
+    {name: 'Potato', isWaste: 'custom', custom: 'depends', explanation: 'if you grow your own then no if you go buy at the store than yes'}
```

Comment:
You are doing something wrong here:
isWaste should be a boolean, 'custom' or 'free'. You are using strings as booleans.
```suggestion
    {name: 'scratch', isWaste: false, explanation: 'Come on! Scratch is awesome and its totally free!'}
    {name: 'github', isWaste: 'free'}
    {name: 'github teams', isWaste: 'custom', explantion: 'it has free a option and if you pick that the no but if you pay for it yes'}
    {name: 'github PRO', isWaste: true}
    {name: 'Potato', isWaste: 'custom', custom: 'depends', explanation: 'if you grow your own then no if you go buy at the store than yes'}
```